docs/HOOKS.md

```diff
@@ -926,3 +930,67 @@ This function derives the visibility of the typing indicator via:
 
 -  `typingAnimationDuration` value specified in style options, in milliseconds
 -  Values from the `useLastTypingAt` hook
+
+## Telemetry
+
+### `useTrackDimension`
+
+```js
+useTrackDimension(): (name: string, data?: string) => void
+```
+
+This function will add, update, or remove a dimension from telemetry. If `undefined` is passed to `data`, the dimension will be removed. This hook will not trigger `onTelemetry` handler.
+
+### `useTrackEvent`
+
+```js
+type EventData =
+  number |
+  string |
+  { [key: string]: number | string };
+
+useTrackEvent():
+  (name: string, data?: EventData) => void &
```

Comment:
Should the return type be `void | { ... }`

Reply:
Fixed, thanks!

```ts
useTrackEvent(): {
  (name: string, data?: EventData): void;
  debug: (name: string, data?: EventData) => void;
  error: (name: string, data?: EventData) => void;
  info: (name: string, data?: EventData) => void;
  warn: (name: string, data?: EventData) => void;
}
```

> Wondered, when should we use `&` in type, need some read ups.